recipes-kernel/linux/files/bcm2712-raspberrypi5-domd.dts

```diff
@@ -88,8 +77,8 @@
 			 * msi-controller@130000 has failed - no IRQs received by macb
 			 * 1f00100000.ethernet. Reason unknown.
 			 */
-//			msi-parent = <&mip0>;
-			msi-parent = <&pciex>;
+			msi-parent = <&mip0>;
+//			msi-parent = <&pciex>;
```

Comment:
Could you please remove this line altogether?

Reply:
done